README.md

```diff
@@ -1180,13 +1180,12 @@ Start an interactive shell.
 Start an interactive shell.
 
 USAGE
-  $ fauna shell [DBNAME] [--endpointURL <value>] [--timeout
-    <value>] [--secret <value>] [--endpoint <value>] [--file <value>] [--stdin]
-    [--output <value>] [--format json|json-tagged|shell] [--version 4|10]
-    [--typecheck]
+  $ fauna shell [SCOPE] [--endpointURL <value>] [--timeout <value>]
```

Comment:
Update the README?

Reply:
yeah, I just updated the README to match the new `--help` output